docs/src/index.md

```diff
@@ -55,3 +55,20 @@ will trigger a download dialog to `~/.julia/datadeps/MNIST`. To
 overwrite this on a case by case basis, it is possible to specify
 a data directory directly in `traindata(dir = <directory>)` and
 `testdata(dir = <directory>)`.
+
+## Preview
+
+For image dataset, [`MosaicViews`](https://github.com/JuliaArrays/MosaicViews.jl) provides a very simple interface to display a list
+of images.
```

Comment:
The fact that MosaicViews is not directly used in the code below is a bit confusing. 
Maybe we can just point to the `mosaic` function from `ImageCore`.